docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -135,10 +135,11 @@ Each component in a single devfile must have a unique name and use one of the ob
 
 . For each component, define an unique value for the mandatory `name` attribute.
 
-. For each component, define a mandatory type of one of the following types: `kubernetes`, `container` or `volume`.
+. For each component, define a mandatory type of one of the following types: `kubernetes`, `container`, or `volume`.
 
-. xref:adding-kubernetes-component-to-a-devfile.adoc[]
-. xref:adding-container-component-to-a-devfile.adoc[]
+. xref:adding-a-kubernetes-or-openshift-component-to-a-devfile.adoc[]
+. xref:adding-a-container-component-to-a-devfile.adoc[]
+. xref:adding-a-volume-component-to-a-devfile.adoc[]
 . xref:specifying-persistent-storage.adoc[]
 . xref:limiting-resources-usage.adoc[]
 . xref:defining-environment-variables.adoc[]
```

Comment:
Suggestion: L140 to L145: Modify the heading texts by removing the gerunds (Add, Specify, Limit, and Define). Example (L144): . xref:limiting-resources-usage.adoc[Limit resources usage]. This way the content of the steps and the writing style are in sync with the first 3 steps of the procedure (parallelism).